lib/cisco_node_utils/fabricpath_global.rb

```diff
@@ -0,0 +1,439 @@
+#
+# NXAPI implementation of Fabricpath Global class
+#
+# November 2015, Deepak Cherian
+#
+# Copyright (c) 2015 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative 'node_util'
+
+module Cisco
+  # node_utils class for fabricpath_global
+  class FabricpathGlobal < NodeUtil
+    attr_reader :name
+
+    def initialize(name, instantiate=true)
+      fail TypeError unless name.is_a?(String)
+      fail ArgumentError unless name == 'default'
+      @name = name.downcase
+      @set_params = {}
+
+      create if instantiate
+    end
+
+    def self.globals
```

Comment:
I remember now why I didn't like this... it's because the feature checker returns four possible symbol states (as the fex feature does as well). 
It would have been better to just have it return true/false. If it's not both installed and enabled then it's false. All the extra symbol checking just complicates things and it's not going to hurt to re-enter the install command if it's already installed.
Anyway, what you have is fine but feel free to rework it. :-)